packages/config/config/devices/0x0346/keypad.json

```diff
@@ -0,0 +1,387 @@
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
+			]
+		},
+		"14": {
+			"label": "Proximity Distance",
+			"unit": "cm",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 100,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"15": {
+			"label": "Extended Battery Life LED Brightness Adjustment",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 50,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"16": {
+			"label": "Reduced Battery Life LED Brightness Adjustment",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 65,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"17": {
+			"label": "Fast Blink Rate",
+			"unit": "0.1 sec",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"18": {
+			"label": "Slow Blink Rate",
+			"unit": "0.5 sec",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 2,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"19": {
+			"label": "Supervision Report Timeout",
+			"valueSize": 2,
+			"minValue": 500,
+			"maxValue": 5000,
+			"defaultValue": 1500,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"20": {
+			"label": "Number of Re-transmissions Attempts",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"21": {
+			"label": "Wait Period Between Re-transmissions Attempts. settings",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 60,
+			"defaultValue": 5,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"22": {
+			"$if": "productId === 0x0202",
+			"label": "Language",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 31,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "US English",
+					"value": 0
+				},
+				{
+					"label": "UK English",
+					"value": 1
+				},
+				{
+					"label": "French",
+					"value": 2
+				},
+				{
+					"label": "German",
+					"value": 3
+				},
+				{
+					"label": "Italian",
+					"value": 4
+				},
+				{
+					"label": "Spanish",
+					"value": 5
+				},
+				{
+					"label": "Dutch",
+					"value": 6
+				},
+				{
+					"label": "Danish",
+					"value": 7
+				},
+				{
+					"label": "Norwegian",
+					"value": 8
+				},
+				{
+					"label": "Swedish",
+					"value": 9
+				},
+				{
+					"label": "Finnish",
+					"value": 10
+				},
+				{
+					"label": "Arabic",
+					"value": 11
+				}
+			]
+		},
+		"23": {
```

Comment:
This could probably become a subfield where each bit is if a specific country is enabled or not. But because I can't test it I felt like it is probably best to leave it general.

Reply:
We need to split these out if it's a partial parameter. What do they report?

Reply:
I don't have a EU version of the keypad so I can't test it. If you prefer to leave out the EU part of now I can do that. I only included them because I thought it was better to have something than nothing.